guides/cloud/aws_sandbox.md

```diff
@@ -0,0 +1,42 @@
+# Made Tech AWS Sandbox
+
+## Overview
+
+Made Tech have an AWS Sandbox account for individuals to use for personal development and testing. We run [AWS SSO](https://aws.amazon.com/single-sign-on/) to simplify and secure the IAM aspect of accessing this account.
+
+## Usage Policy
+
+This AWS account should only be used for temporary short lived test type projects and any resources are subject to being destroyed on a nightly or weekly basis. Individuals are responsible for cleaning up any resources they create and could be held liable for any ludicrous bills incurred by accidentally leaving giant Elasticsearch clusters running.
```

Comment:
Do we have any billing alarms/alerts set up to help as a safeguard?

Reply:
Yup we do, this statement is just to remind people to be responsible for the resources they create, and clean up after themselves.